subsys/mgmt/suitfu/Kconfig

```diff
@@ -16,6 +16,12 @@ config MGMT_SUITFU
 	select MCUMGR_SMP_CBOR_MIN_ENCODING_LEVEL_3
 
 
+config MGMT_SUITFU_INITIALIZE_SUIT
+	bool "Initialize the SUIT DFU library"
+	default y
```

Comment:
there is no if in this file to ensure the suitfu Kconfig is selected to gate these options

Reply:
That's strange.. 
Merged `menu` and `config` into `menuconfig`.